java/code/src/com/redhat/rhn/frontend/dto/VirtualSystemOverview.java

```diff
@@ -67,7 +68,8 @@ public VirtualSystemOverview(Tuple tuple) {
             stateLabel = getTupleValue(tuple, "state_label", String.class).orElse(null);
             vcpus = getTupleValue(tuple, "vcpus", Number.class).map(Number::longValue).orElse(0L);
             memory = getTupleValue(tuple, "memory", Number.class).map(Number::longValue).orElse(0L);
-            hostServerName = getTupleValue(tuple, "host_server_name", String.class).orElse(null);
+            virtualHostName = getTupleValue(tuple, "virtual_host_name", String.class).orElse(null);
```

Comment:
Why changing  the field name?

Reply:
yeah, you're right, it goes in line with the original names. already rolled it back